xml/ha_config_basics.xml

```diff
@@ -2263,17 +2264,58 @@ monitor_0     interval=5s timeout=20s</screen>
     </para>
    </note>
    <para>
-    Apart from detecting the minimal requirements, the &hasi; also allows
-    to monitor the current utilization via the
+    Apart from detecting the minimal requirements, you can also
+    monitor the current utilization via the
     <systemitem>VirtualDomain</systemitem> resource agent. It detects CPU
     and RAM use of the virtual machine. To use this feature, configure a
     resource of the following class, provider and type:
     <literal>ocf:heartbeat:VirtualDomain</literal>. The following instance
-    attributes are available: <varname>autoset_utilization_cpu</varname> and
-    <varname>autoset_utilization_hv_memory</varname>. Both default to
-    <literal>true</literal>. This updates the utilization values in the CIB
-    during each monitoring cycle.
+    attributes are available:
    </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      <varname>autoset_utilization_cpu</varname>
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <varname>autoset_utilization_hv_memory</varname> (for Xen) or
+      <varname>autoset_utilization_host_memory</varname> (for KVM)
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    These attributes default to <literal>true</literal>. This updates the utilization
+    values in the CIB during each monitoring cycle. For more information, see
+    <command>crm ra info VirtualDomain</command>.
+   </para>
+   <note>
+    <title><literal>hv_memory</literal> and <literal>host_memory</literal></title>
+    <para>
+     In the <systemitem>NodeUtilization</systemitem> and <systemitem>VirtualDomain</systemitem> resource agents, <literal>hv_memory</literal> and <literal>host_memory</literal> both default to <literal>true</literal>. However, Xen only requires <literal>hv_memory</literal>, and KVM only requires <literal>host_memory</literal>. To avoid confusion, we recommend disabling the unnecessary attributes. For example:
+    </para>
+    <example xml:id="ex-kvm-disable-hv-memory">
+     <title>Creating resource agents for KVM with <literal>hv_memory</literal> disabled</title>
+     <screen>primitive p_nu NodeUtilization \
+      params utilization_hv_memory=false \
+      op monitor timeout=20s interval=60
+
+primitive p_vm VirtualDomain \
+      params autoset_utilization_hv_memory=false \
+      op monitor timeout=30s interval=10s</screen>
+    </example>
+    <example xml:id="ex-xen-disable-host-memory">
+     <title>Creating resource agents for Xen with <literal>host_memory</literal> disabled</title>
+     <screen>primitive p_nu NodeUtilization \
+      params utilization_host_memory=false \
+      op monitor timeout=20s interval=60
+
+primitive p_vm VirtualDomain \
+      params autoset_utilization_host_memory=false \
+      op monitor timeout=30s interval=10s</screen>
+    </example>
+   </note>
```

Comment:
Such an embarrassing but very thoughtful note :-) I don't have a better idea though. Thanks, @tahliar and @zzhou1 !

Reply:
Thank you @tahliar to rectify this situation in DOC !